datalad/customremotes/base.py

```diff
@@ -66,131 +63,6 @@ def get_function_nargs(f):
         return len(argspec.args) - 1
 
 
-class AnnexExchangeProtocol(ProtocolInterface):
```

Comment:
bye bye my Dear friend, you were a good help!  I can only hope that I will not miss you, in log and Joey we trust!